### DISCUSSION TITLE
Multilingual Mukurtu site

### DISCUSSION THREAD

**Discussion by MonicaGalassi** [MukurtuCMS/Mukurtu-CMS#26] [Feature Enhancement]
@michael-wynne-wsu I am using Mukurtu with institutions that don't speak English as first language and it has been a challenge to use mukurtu in that context. I imagine an expert could translate the interface but I wasn't able to. So I have used a drupal site and linked it to Mukurtu (it works but it's not ideal). I was thinking of an integration with a multilingual software out of the box would be incredibly helpful.

**Comment by michael-wynne-wsu** [comment]
Mukurtu 4 should be fully compatible with Drupal's stock localization/translation options. We still need to test to confirm (and go about the best way to translate the site elements, ideally offer multilingual packages provided by users, etc)... So this will be better, but if you have any thoughts about preferred ways to toggle between site languages and/or view multiple languages within a site, any input would be great.
